javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+var width = 50;
+var height= 10;
+var area = width * height;
+console.log("value of area:", area);
+console.log("type of area:", typeof area)
+var bill = 10;
+var payment= 20;
+var change= payment- bill;
+console.log("value of change:", change);
+console.log("type of change:", typeof change);
+var quizzes = 60;
+var midterm= 60;
+var final= 70;
+var grade= (quizzes+midterm+final) / 3;
+console.log("value of grade:", grade);
+console.log("type of grade:", typeof grade);
+var firstName = "Mohamed";
+var lastName= "Elmessiry"
+var fullName= firstName+" "+ lastName;
+console.log("value of fullName:", fullName);
+console.log("type of fullname:", typeof fullName);
+var pH = 10;
+var isAcidic= (pH < 7);
+console.log("value of isAcidic:",isAcidic);
+console.log("type of isAcidic:", typeof isAcidic);
+var headCount = 50;
+var isSparta= headCount === 300;
+console.log("value of isSparta:", isSparta);
+console.log("type of isSparta:", typeof isSparta);
+var motto = fullName +" "+"is the GOAT";
```

Comment:
Please put a space before and after operators

Reply:
It's weird, as I pushed the exercise when it's spaced out, will push again :)